Jenkinsfile.public

```diff
@@ -209,6 +209,21 @@ pipeline {
                     }
                 }
 
+                stage('GraphQL Endpoints') {
+                    environment {
+                        // use different hostname to avoid port conflicts
+                        STORJ_NETWORK_HOST4 = '127.0.0.7'
+                        STORJ_NETWORK_HOST6 = '127.0.0.7'
+                        SATELLITE_0_ADDR='127.0.0.7:10002'
+                        STORJ_SIM_POSTGRES = 'postgres://postgres@localhost/teststorj7?sslmode=disable'
+                        }
+                    steps {
+                        // sh 'docker pull postman/newman:alpine'
```

Comment:
note, we do not want docker inside docker

Reply:
Yeah I agree and as discussed I will install postman/newman via npm so that we can get some test coverage running in the pipeline. Afterwards, I will start working on implementing these tests via go as they are very basic in nature and as a team we want to reduce the amount of tools that we will need to manage going forward.